### ISSUE TITLE
openulmus.org -> Blast Emails are marked as Spam 

### ISSUE THREAD

**Issue by cleverington** [elmsln/openulmus.org#41]
If we're going official, which we should, and planning on offering white-label style services for dontation-returns, we're also going to need to get our emails to _not_ be flagged as Spam by the SpamBots.

![screen shot 2017-11-27 at 7 27 41 pm](https://user-images.githubusercontent.com/1015474/33297891-13d82722-d3a9-11e7-9d93-38aa387b41c2.png)

**Comment by mmilutinovic013** [comment]
I'll self-assign and mark as a bug in the ELMS:LN issue queue.

**Comment by mmilutinovic013** [comment]
JK I'll do it internal here.

**Comment by mmilutinovic013** [comment]
Dear [user:name],

A request has been made to reset the password for your account on [site:name].openulmus.org. You may now log in by clicking this link or copying and pasting it to your browser:

[user:one-time-login-url]

This link can only be used once to log in and will lead you to a page where you can set your password. It will expire after one day and nothing will happen if it's not used.

Best wishes!
The Open Ulmus Team

**Comment by mmilutinovic013** [comment]
Updated it -- wanna test with your inbox @cleverington ?

**Comment by cleverington** [comment]
@mmilutinovic1313 

Nope. We'll need to research "How to become a confirmed site-owner" or something. 

It isn't on Drupal's side (I think) it is on either the mail-settings, Apache, Google (or all of the above).

**Comment by mmilutinovic013** [comment]
Okay good to know - I thought it might have been the content. It comes from @btopro 's OU email so maybe we should have an admin.OU email instead.

**Comment by mmilutinovic013** [comment]
I'll post additional findings here and patch

**Comment by cleverington** [comment]
## Update

Reference Notes -> https://groups.drupal.org/node/154874 

There will require some steps to get off 'White Listed', which will likely require 1 - 4 hours of dev-time (at a guess).
